packages/postgres-database/setup.py

```diff
@@ -19,7 +19,7 @@ def read_reqs(reqs_path: Path):
 install_requirements = read_reqs(current_dir / "requirements" / "_base.in")
 
 # Strong dependencies
-migration_requirements = read_reqs(current_dir / "requirements" / "_migration.txt")
+migration_requirements = read_reqs(current_dir / "requirements" / "_migration.in")
```

Comment:
is this correct? from .txt to .in?

Reply:
yes. in libraries (i.e. all modules under ``packages`` folder) we do not want to pin dependencies but only add constraints (if any)